nixos/modules/services/mail/davmail.nix

```diff
@@ -0,0 +1,43 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.davmail;
+in
+  {
+    options.services.davmail = {
+      enable = mkEnableOption ''
+        DavMail is a POP/IMAP/SMTP/Caldav/Carddav/LDAP exchange gateway
+        allowing users to use any mail/calendar client with an Exchange server.
+      '';
+
+      package = mkOption {
+        default = pkgs.davmail;
+        type = types.package;
+        defaultText = "pkgs.davmail";
+        description = "Which davmail package to use.";
```

Comment:
If there's no alternate package for davmail or any other good reason, I prefer not to have such a package option, see https://github.com/NixOS/nixpkgs/issues/50476